crates/pallet-grandpa-finality-verifier/src/lib.rs

```diff
@@ -59,16 +59,15 @@ pub struct InitializationData {
 
 #[frame_support::pallet]
 pub mod pallet {
-    use crate::chain::Chain;
-    use crate::grandpa::{
-        find_forced_change, find_scheduled_change, verify_justification, AuthoritySet,
+    use crate::{
+        chain::Chain,
+        grandpa::{find_forced_change, find_scheduled_change, verify_justification, AuthoritySet},
+        InitializationData,
     };
```

Comment:
Actually we typically don't do nested imports, combining only same level imports together :confounded:

Reply:
Should we just set rustfmt [option](https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#Module%5C%3A) for that.

Reply:
okay. no opinion here. rustfmt route looks cleaner IMO

Reply:
Let's not do that in this PR though. It'll get MUCH bigger for no reason and will likely cause A LOT of conflicts with https://github.com/subspace/subspace/pull/329 for instance. We need to pick a quiet time in terms of global changes and do the switch. Or update crate by crate as we work on them before enforcing global rules.

Reply:
Sure thing. It is just that how to prevent this kind of comments once and for all.

Reply:
We need to review a bunch of rules there. There are rules for automatically formatting comments to not exceed a specific length for instance.

Reply:
Sorry, link for option is broken at top. I meant this option:
https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#imports_granularity

```toml
imports_granularity = "Module"
```